mint-005.md

```diff
@@ -0,0 +1,165 @@
+
+
+##### Valid Layer 3 Spend Conditions
+
+| Spending Scenario | PAK_1 | PAK_2 | PAK_3 | SAK | BIP-113 time greater than  (between_epoch_timestamp) |
+|-------------------|:-----:|:-----:|:-----:|:---:|:----------------------------------:|
+| Scenario 19       | ✅     | ✅     |       | ✅  | ✅                                  |
+| Scenario 20       | ✅     |       | ✅     | ✅  | ✅                                  |
+| Scenario 21       |       | ✅     | ✅     | ✅  | ✅                                  |                                |
+
+
+#### Layer 4:
+| Sovereign Recovery Path | $RK_1$ | $RK_2$ | $RK_3$ | Network BIP-113 time greater than `largest_epoch_timestamp` |
+|:-----------------------:|:------:|:------:|:------:|:-------:|
+| 2 of 3 RKs AND after BIP-113 time is greater than `largest_epoch_timestamp` | ![RK1](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![RK2](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![RK3](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/key.png) | ![Timelock](https://raw.githubusercontent.com/bitcoincore-dev/miniscript-templates/main/assets/unlock.png) |
+
+##### Valid Layer 4 Spend Conditions
+
+| Spending Scenario | RK_1 | RK_2 | RK_3 | Timelock (largest_epoch_timestamp) |
+|-------------------|:----:|:----:|:----:|:----------------------------------:|
+| Scenario 22       | ✅    | ✅    |      | ✅                                  |
+| Scenario 23       | ✅    |      | ✅    | ✅                                  |
+| Scenario 24       |      | ✅    | ✅    | ✅                                  |
+
+
+---
+
+# Example Miniscript Output Descriptor
+
+For this example, the `smallest_epoch_timestamp` is: 1672531200 (Jan 1 2023, midnight gmt), the `between_epoch_timestamp` is: 1673740800 and `largest_epoch_timestamp` is: 1675209600 (Feb 1 2023, midnight gmt)
+
+- MINT-005 Output Descriptor: 
+<code>wsh(andor(multi(2,$PAK_1$,$PAK_2$,$PAK_3$),or_i(and_v(v:pkh($SAK$),after(`between_epoch_timestamp`)),thresh(2,pk($PK_1$),s:pk($PK_2$),s:pk($PK_3$),snl:after(`smallest_epoch_timestamp`))),and_v(v:thresh(2,pkh($RK_1$),a:pkh($RK_2$),a:pkh($RK_3$)),after(`larget_epoch_timestamp`))))</code>
+
+
+- Source Policy (FOR REFERENCE PURPOSES ONLY):
+<code>"or(99@and(thresh(2,pk($PAK_1$),pk($PAK_2$),pk($PAK_3$)),or(99@thresh(2,pk($PK_1$),pk($PK_2$),pk($PK_3$),after(`smallest_epoch_timestamp`)),and(pk($SAK$),after(`between_epoch_timestamp`)))),and(thresh(2,pk($RK_1$),pk($RK_2$),pk($RK_3)),after(`largest_epoch_timestamp`)))"</code>
+
+
+## Layer 1 Example Spend
+
+Signed by: $PK_1$, $PK_2$, $PAK_1$, $PAK_2$
+
+[Reference Testnet
+Transaction](https://mempool.space/signet/tx/2836d6af6b5c4bb01e926391f64771fb333193676040b24d4236ba0bb89a7008)
+
+## Layer 2 Example Spend 
```

Comment:
In 7bebe7efd503756903ccb9efc65e1e681a5ebb4e:

trailing whitespace

Reply:
Updated